assets/js/theme/account.js

```diff
@@ -5,6 +5,7 @@ import Wishlist from './wishlist';
 import validation from './common/form-validation';
 import stateCountry from './common/state-country';
 import { classifyForm, Validators, insertStateHiddenField } from './common/form-utils';
+import swal from 'sweetalert2';
```

Comment:
You may choose to not import the vendor library directly. Instead, create a facade so that it's easier to change its implementation later.